homeassistant/components/trend/const.py

```diff
@@ -0,0 +1,14 @@
+"""Constant values for Trend integration."""
+DOMAIN = "trend"
+
+ATTR_ATTRIBUTE = "attribute"
+ATTR_GRADIENT = "gradient"
+ATTR_MIN_GRADIENT = "min_gradient"
+ATTR_INVERT = "invert"
```

Comment:
Consider sorting these ATTR and CONF values.

https://developers.home-assistant.io/docs/development_guidelines

> Constants and the content of lists and dictionaries should be in alphabetical order.

Reply:
Thanks! Done